source/features/action-used-by-link.tsx

```diff
@@ -7,7 +7,7 @@ import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 
 function getActionURL(): URL {
-	const actionRepo = $('aside a:has(.octicon-repo)')
+	const actionRepo = $('[data-testid="resources"] > ul a:has(.octicon-repo)')
```

Comment:
```suggestion
	const actionRepo = $('a:has(.octicon-repo)', side)
```

You can pass the element selected by the observer instead